api/v1/configurationpolicy_types.go

```diff
@@ -282,7 +296,7 @@ type ConfigurationPolicySpec struct {
 
 // ComplianceState reports the observed status from the definitions of the policy.
 //
-// +kubebuilder:validation:Enum=Compliant;Pending;NonCompliant;Terminating
+// +kubebuilder:validation:Enum=Compliant;Pending;NonCompliant;Terminating;UnknownCompliancy
```

Comment:
@JustinKuli there was a bug in the code during development and the status update was failing because `UnknownCompliancy` was not a valid value. I just added it here since I thought it was an oversight since it is a value the status update code can set.

Is it your preference that I remove it? It should not be possible to get this value in ConfigurationPolicy but I wondered if having a status of UnknownCompliancy is better than the status update failing if a bug is introduced.

Reply:
We can keep it for completeness. Do you mean that it can be set by status-sync or something else in the framework?

If we were making it from scratch, my preference would just be to have an empty value imply an unknown compliance, as opposed to a special value. But if other things already use UnknownCompliancy, then it's not worth changing everywhere, and it should at least be mentioned here for user clarity.

Reply:
I'll just remove it.